### DISCUSSION TITLE
Dockerize GrumPHP

### DISCUSSION THREAD

**Discussion by albertocubeddu** [phpro/grumphp#1083] [Ideas]
GrumPHP is a fantastic tool, and the combination with docker makes it extremely powerful. 

We've done an article about it: https://medium.com/creditorwatch/scale-teams-with-grumphp-6c582e0b09d4
and you can find a PoC in my Github: https://github.com/albertocubeddu/grumPHP-docker

Kudos to phPro! We're immensely impressed with the outstanding performance, exceptional efficiency, and remarkable user-friendliness delivered by your work!

**Comment by veewee** [comment]
Looks nice, thanks for sharing!

Another approach would be to create a convention and optionally to run it on your app's php container directly. That might safe you some problems like the need of a specific extensions etc 

More info
* https://github.com/phpro/grumphp/blob/master/doc/conventions.md
* https://github.com/phpro/symfony-conventions
